frontend/src/app/modules/common/tabs/tab-badges/tab-count.component.ts

```diff
@@ -0,0 +1,12 @@
+import { ChangeDetectionStrategy, Component, Input, OnChanges, OnInit } from "@angular/core";
+import { Observable } from "rxjs";
+
+@Component({
+  selector: 'op-tab-count',
+  templateUrl: './tab-count.html',
+  styleUrls: ['./tab-count.sass'],
```

Comment:
`.component.` for both html template and sass would be nice. I use it to separate template-specific sass from general BEM/util stuff.